src/ui/components/pages/assets/helpers.ts

```diff
@@ -5,9 +5,39 @@ const explorerUrls = new Map([
   ['custom', 'wavesexplorer.com/custom'],
 ]);
 
-export function getTxLink(txId: string, networkCode: string): string {
+export function getTxHistoryLink(networkCode: string, address: string): string {
+  const explorer = explorerUrls.get(
+    explorerUrls.has(networkCode) ? networkCode : 'custom'
+  );
+  return `https://${explorer}/address/${address}/tx/`;
+}
+
+export function getTxDetailLink(networkCode: string, txId: string): string {
   const explorer = explorerUrls.get(
     explorerUrls.has(networkCode) ? networkCode : 'custom'
   );
   return `https://${explorer}/tx/${txId}`;
 }
+
+export function getAssetDetailLink(
+  networkCode: string,
+  assetId: string
+): string {
+  const explorer = explorerUrls.get(
+    explorerUrls.has(networkCode) ? networkCode : 'custom'
+  );
+  return `https://${explorer}/assets/${assetId}`;
+}
+
+export const colors = {
+  white: '#FFFFFF',
+  basic200: '#DAE1E9',
+  basic500: '#9BA6B2',
+  submit200: '#BACAF4',
+  submit400: '#1F5AF6',
+  out: '#FFAF00',
+  in: '#81C926',
+};
```

Comment:
Not sure about `out` and `in` as being generic enough, but other colors are the same as in `colors.styl`. May be use css custom properties for them? With something like this in `app.styl`:

```css
:root {
  --color-white: #fff;
  --color-basic200: #dae1e9;
  /* etc. */
}
```

And then it can be used as `var(--color-basic200)` in css or for coloring svgs.